### PR TITLE
Toddbell/cpp hardcode fix

### PR DESCRIPTION
**Pull request by toddbellMSFT**

### CODE REVIEW

File: targets/XPlatCppSdk/source/LibUnitTest/PlayFabEntityTest.cpp [hunk 1]
```diff
@@ -216,6 +216,7 @@ namespace UnittestRunner
         }
         static void SetObjectsCallback(const SetObjectsResponse& result, void*)
```
Comment (pgilmorepf) [inline targets/XPlatCppSdk/source/LibUnitTest/PlayFabEntityTest.cpp]: Don't do the UNREFERENCED_VARIABLE(result); thing below.
Fix result here by removing the result keyword.

File: targets/XPlatCppSdk/source/LibUnitTest/stdafx.h [hunk 2]
```diff
@@ -3,6 +3,8 @@
 // Headers for CppUnitTest
 #include <CppUnitTest.h>
 
+#define UNREFERENCED_VARIABLE(X) (void) (X);
```
Comment (pgilmorepf) [inline targets/XPlatCppSdk/source/LibUnitTest/stdafx.h]: Don't do this

File: targets/XPlatCppSdk/source/LibUnitTest/PlayFabEntityTest.cpp (hunk 1 above)
Comment (pgilmorepf) [reply targets/XPlatCppSdk/source/LibUnitTest/PlayFabEntityTest.cpp]: So literally:
static void SetObjectsCallback(const SetObjectsResponse&, void*)
That's correct syntax and it'll remove that warning/error.

File: targets/XPlatCppSdk/source/cppsdk/XPlatCppWindows.vcxproj.filters.ejs [hunk 3]
```diff
@@ -17,45 +17,22 @@
     </Filter>
   </ItemGroup>
   <ItemGroup>
-    <ClInclude Include="include\playfab\PlayFabEntityDataModels.h">
-      <Filter>Header Files\playfab</Filter>
-    </ClInclude>
-    <ClInclude Include="include\playfab\PlayFabHttp.h">
```
Comment (pgilmorepf) [inline targets/XPlatCppSdk/source/cppsdk/XPlatCppWindows.vcxproj.filters.ejs]: You should not have removed this one.
Are you sure this compiled and ran?
You can't run the unit tests, but you CAN run the Windows Example Project.

Comment (toddbellMSFT) [reply targets/XPlatCppSdk/source/cppsdk/XPlatCppWindows.vcxproj.filters.ejs]: It's still there (check out line 30).

Comment (pgilmorepf) [reply targets/XPlatCppSdk/source/cppsdk/XPlatCppWindows.vcxproj.filters.ejs]: resolved. Thanks. Just missed it.

**Pull request merged**